scripts/build/build.js

```diff
@@ -1,108 +1,230 @@
-#!/usr/bin/env node
-
 "use strict";
 
 const path = require("path");
-const pkg = require("../../package.json");
-const formatMarkdown = require("../../website/playground/markdown");
-const parsers = require("./parsers");
-const shell = require("shelljs");
+const { rollup } = require("rollup");
+const webpack = require("webpack");
+const resolve = require("rollup-plugin-node-resolve");
+const commonjs = require("rollup-plugin-commonjs");
+const nodeGlobals = require("rollup-plugin-node-globals");
+const json = require("rollup-plugin-json");
+const replace = require("rollup-plugin-replace");
+const uglify = require("rollup-plugin-uglify");
+const babel = require("rollup-plugin-babel");
+const nativeShims = require("./rollup-plugins/native-shims");
+const executable = require("./rollup-plugins/executable");
+
+const Bundles = require("./bundles");
+const util = require("./util");
+
+const EXTERNALS = [
+  "assert",
+  "buffer",
+  "constants",
+  "crypto",
+  "events",
+  "fs",
+  "module",
+  "os",
+  "path",
+  "stream",
+  "url",
+  "util",
+  "readline",
+
+  // See comment in jest.config.js
+  "graceful-fs"
+];
+
+function getBabelConfig(bundle) {
+  const config = {
+    babelrc: false,
+    presets: [],
+    plugins: ["external-helpers"]
+  };
+  if (bundle.type === "core") {
+    config.plugins.push(require("./babel-plugins/transform-eval-require"));
+  }
+  if (bundle.transpile) {
+    config.presets.push(["es2015", { modules: false }]);
```

Comment:
I know we were using es2015 before, but we should change this to use `babel-preset-env` instead of `babel-preset-es2015`, as es2015 is deprecated.

Reply:
Yeah I was gonna do it in a follow up since there are a LOT of changes already, but I guess it doesn't matter much now.